composer.json

```
@@ -23,12 +23,19 @@
         "ext-sqlite3": "*",
         "ext-xmlreader": "*"
     },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^3.4.0"
```

```suggestion
        "friendsofphp/php-cs-fixer": "^3.91.0"
```

While I love 3.5.0 Si!, it's coming back from 2021.